consensus/module.go

```diff
@@ -116,7 +138,39 @@ func Create(cfg *genesis.Config, genesis *genesis.GenesisState) (modules.Consens
 	return m, nil
 }
 
-func (m *consensusModule) Start() error {
+func (m *ConsensusModule) InitConfig(pathToConfigJSON string) (config modules.ConfigI, err error) {
+	data, err := ioutil.ReadFile(pathToConfigJSON)
```

Comment:
Lines 142 - 150 (8 lines) will need to be copy-pasted twice for every single module that we create. This triggers my "allergies" that we're doing something wrong.

I think we should create a ticket to improve this but wanted to see what you think.

Reply:
We can move the ioutil to a shared utility file if you'd prefer - but I don't think this is a big deal